bindata/network/ovn-kubernetes/006-ovnkube-config.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# The config file used by ovnkube is passed as a configmap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovnkube-config
+  namespace: openshift-ovn-kubernetes
+data:
+  config:   |-
+    [default]
+    mtu="{{.MTU}}"
+
+    [kubernetes]
+    service-cidr="{{.OVN_service_cidr}}"
+    cluster-subnets={{.K8S_cluster_subnets}}
+    ovn-config-namespace="{{.OVN_namespace}}"
+
+    [logging]
+    loglevel=2
+
+    [ovnnorth]
+    client-privkey="{{.OVNKUBE_nb_privkey}}"
```

Comment:
For all the cert variables, let's just remove them from the config for now, since the default is empty anyway. When we figure out what needs to be done for SSL we can add them back.

Reply:
Indeed, we should not be putting secrets in configmaps.

The typical solution is to write them on disk via Secrets, and just configure the path to them via config files.

Reply:
@squeed client-privkey is a defined part of the config file. 
@dcbw @squeed, eliminated [ovnnorth] and [ovnsouth] sections. When ssl is supported they can be added back.